File: Data Structures & Algorithms/two-integer-sum-ii/submission-1.py
class Solution:
    def twoSum(self, numbers: List[int], target: int) -> List[int]:
        myMap = defaultdict(int)
        for i in range(len(numbers)):
            find = target - numbers[i]
            if myMap[find]:
                return [myMap[find],i +1]
            myMap[numbers[i]]= i + 1
        return []

File: Data Structures & Algorithms/two-integer-sum/submission-1.java
class Solution {
    public int[] twoSum(int[] nums, int target) {
        if (nums[0] + nums[1] == target){
            return new int[] {0,1};
        }
        for (int i = 0; i < nums.length; ++i){
            for (int j = i + 1; j < nums.length; ++j){
                if (nums[i] + nums[j] == target){
                    return new int[] {i,j};
                }
            }
        }
        return nums;
    }
}
